### DISCUSSION TITLE
Chart widget card not cover entire chart

### DISCUSSION THREAD

**Discussion by hazwan91** [filamentphp/filament#14318] [Help] [answered]
### Package

Widgets

### Package Version

v3.2

### How can we help you?

![Capture](https://github.com/user-attachments/assets/656196b2-f4df-4bb6-8501-3c77e1e87cdb)

my code

```
<?php

namespace App\Filament\Widgets;

use App\Models\ApplicationBitk;
use Carbon\Carbon;
use Filament\Widgets\ChartWidget;

class ApplicationBitksChart extends ChartWidget
{
    protected static ?string $heading = 'Statistik Permohonan';

    protected static ?int $sort = 2;

    protected int | string | array $columnSpan = 'full';

    protected function getData(): array
    {
        $data = $this->getApplicationsPerMonth();
        return [
            'datasets' => [
                [
                    'label' => 'Jumlah Permohonan',
                    'data' => $data['applicationsPerMonth']
                ]
            ],
            'labels' => $data['months']
        ];
    }

    protected function getType(): string
    {
        return 'bar';
    }

    private function getApplicationsPerMonth(): array {
        $now = Carbon::now();
        $applicationsPerMonth = [];
        $months = collect(range(1, 12))->map(function($month) use ($now, &$applicationsPerMonth) {
            $count = ApplicationBitk::whereMonth('created_at', Carbon::parse($now->month($month)->format('Y-m')))->count();
            $applicationsPerMonth[] = $count;

            return $now->month($month)->format('M');
        })->toArray();

        return [
            'applicationsPerMonth' => $applicationsPerMonth,
            'months' => $months
        ];
    }
}

```

as title has stated, please help...
i already did clearing cache using
`php artisan optimize:clear` and `php artisan filament:optimize-clear`

i also did `composer update && npm update && npm run build`
so i pretty sure im running latest version of filament..

i also think that this did not happen on previous version...

**Comment by brunoflins** [comment]
i have the same problem here, just copied the exemple from the documentation 
![image](https://github.com/user-attachments/assets/f9ee4ccb-e85d-4c97-b72d-67ce84fee539)

**Reply by brunoflins** [reply]
notice that the problem is height 8rem in direct in the element that i dont know where is coming from
![image](https://github.com/user-attachments/assets/d3be5aab-f683-4671-90f1-fdc9ed9415ae)

**Comment by leandrocfe** [comment]
Something related to the livewire update

https://github.com/filamentphp/filament/issues/14315#issuecomment-2366878664

**Reply by leandrocfe** [reply, marked as answer]
https://github.com/filamentphp/filament/releases/tag/v3.2.114
_Lock Livewire to max v3.5.6 due to bug introduced #14315._

**Reply by hazwan91** [reply]
Thanks. My chart looks normal again after updated to latest release of filamentphp..
